File: src/main/java/com/courses/testtask/dto/Employee.java
package com.courses.testtask.dto;

import com.fasterxml.jackson.annotation.JsonFormat;

import javax.persistence.*;
import java.time.LocalTime;
import java.util.Date;

@Entity
@Table

public class Employee {


    @Id
    //@GeneratedValue(strategy = GenerationType.AUTO)
    private Long employee_id ;

    @Column
    private String first_name;
    private String last_name;
    private Integer department_id;
    private String job_title;
    private Gender gender;
    //@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd HH:mm:ss")
    private String date_of_birth;

    public Employee(){}

    public String toString(){
        String line = null;
        return line = this.getEmployee_id() + " " + this.getFirst_name() + " " + this.getLast_name();
    }


    public Long getEmployee_id() {
        return employee_id;
    }

    public void setEmployee_id(Long employee_id) {
        this.employee_id = employee_id;
    }

    public String getFirst_name() {
        return first_name;
    }

    public void setFirst_name(String first_name) {
        this.first_name = first_name;
    }

    public String getLast_name() {
        return last_name;
    }

    public void setLast_name(String last_name) {
        this.last_name = last_name;
    }

    public Integer getDepartment_id() {
        return department_id;
    }

    public void setDepartment_id(Integer department_id) {
        this.department_id = department_id;
    }

    public String getJob_title() {
        return job_title;
    }

    public void setJob_title(String job_title) {
        this.job_title = job_title;
    }

    public String getGender() {
        return gender.toString();
    }

    public void setGender(Gender gender) {
        this.gender = gender;
    }

    public String getDate_of_birth() {
        return date_of_birth;
    }

    public void setDate_of_birth(String date_of_birth) {
        this.date_of_birth = date_of_birth;
    }
}

File: src/main/resources/static/js/employee.js
let employee = employee1;

Vue.component('getOneEmployee', {
    props:['employee'],
    data: function (){
        return {
            employee_id: '',
            first_name: '',
            last_name: '',
            department_id:'',
            job_title:'',
            gender:'',
            date_of_birth:''
        }
    },
    template:
        '<div>' +
        '{{ employee.employee_id }}' +
        '{{ employee.first_name }} {{employee.last_name}} ' +
        '{{employee.department_id}} {{employee.job_title}} ' +
        '{{employee.gender}} {{employee.date_of_birth}} ' +
        '</div>'
})


var app1 = new Vue({
    el: '#app1',
    template: '<getOneEmployee :employee1="employee" />',
    data: {
        employee:[]
    }
});
